common/djangoapps/terrain/stubs/lti.py

```diff
@@ -78,7 +78,7 @@ def do_POST(self):
                         'callback_url': self.post_dict.get('lis_outcome_service_url').replace('https', 'http'),
                         'sourcedId': self.post_dict.get('lis_result_sourcedid')
                     }
-                host = os.environ.get('BOK_CHOY_HOSTNAME', self.server.server_address[0])
+                host = os.environ.get(self.server.server_address[0])
```

Comment:
```suggestion
                host = self.server.server_address[0]
```

You're no longer getting anything from the environment so you don't need to call `os.environ.get`.